CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
```

Comment:
Shouldn't need to add this to the changelog as it is meta data :+1:

Reply:
Reverted back

Reply:
Thanks Richard.